src/tslint-cli.ts

```diff
@@ -152,6 +162,12 @@ tslint accepts the following commandline options:
         and verbose. prose is the default if this option is not used. Additonal
         formatters can be added and used if the --formatters-dir option is set.
 
+    --test:
+        Runs tslint on the specified directory and checks if tslint's output matches
+        the expected output. Automatically loads the tslint.json file in the
```

Comment:
how about "... the expected output in .linttest files" (or whatever we end up naming the extension)